src/components/breadcrumb/breadcrumb.stories.js

```diff
@@ -0,0 +1,29 @@
+import { html } from 'lit-html';
+import { docsIntro, wrapWidth } from '../../../.storybook/utils.js';
+import '.';
+
+export default {
+  title: 'custom-elements/vl-breadcrumb',
+  parameters: {
+    controls: { hideNoControlsWarning: true },
+    docs: {
+      description: {
+        component: docsIntro({
+          root: 'breadcrumb',
+          stylesheets: ['breadcrumb'],
```

Comment:
In de docs tab staat er nog steeds dat de component een stylesheet moet importeren om deze te kunnen gebruiken.